site/layouts/_default/examples.html

```diff
@@ -10,6 +10,10 @@
 
     <link rel="canonical" href="{{ .Permalink }}">
 
+    {{ with .Params.robots -}}
+    <meta name="robots" content="{{ . }}">
+    {{- end }}
+
```

Comment:
Please see [my comment here](https://github.com/jekyll/jekyll-seo-tag/issues/100#issuecomment-293764634) before adding this. I never did hear back from @benbalter or @pathawks on their opinions before the issue was auto-closed. Hope you guys don't mind the ping. :)

Reply:
I don't mind personally using this because it will be used rarely anyway.